src/test/lrsql/admin/protocol_test.clj

```diff
@@ -91,176 +91,177 @@
         sys' (component/start sys)
         lrs  (:lrs sys')
         ds (-> lrs :connection :conn-pool)]
-    (testing "Admin account insertion"
-      (is (-> (adp/-create-account lrs test-username test-password)
-              :result
-              uuid?))
-      (is (-> (adp/-create-account lrs test-username test-password)
-              :result
-              (= :lrsql.admin/existing-account-error))))
-    (testing "Admin account get"
-      (let [accounts (adp/-get-accounts lrs)]
-        (is (vector? accounts))
-        (is (some (fn [acct] (= (:username acct) test-username)) accounts))))
-    (testing "Admin account authentication"
-      (is (-> (adp/-authenticate-account lrs test-username test-password)
-              :result
-              uuid?))
-      (is (-> (adp/-authenticate-account lrs test-username "badPass")
-              :result
-              (= :lrsql.admin/invalid-password-error)))
-      (is (-> (adp/-authenticate-account lrs "foo" "bar")
-              :result
-              (= :lrsql.admin/missing-account-error))))
-    (testing "Admin account existence check"
-      (let [account-id (-> (adp/-authenticate-account lrs
-                                                      test-username
-                                                      test-password)
-                           :result)]
-        (is (adp/-existing-account? lrs account-id)))
-      (let [bad-account-id #uuid "00000000-0000-4000-8000-000000000000"]
-        (is (not (adp/-existing-account? lrs bad-account-id)))))
-    (testing "Admin password update"
-      (let [account-id   (-> (adp/-authenticate-account lrs
+    (try
+      (testing "Admin account insertion"
+        (is (-> (adp/-create-account lrs test-username test-password)
+                :result
+                uuid?))
+        (is (-> (adp/-create-account lrs test-username test-password)
+                :result
+                (= :lrsql.admin/existing-account-error))))
```

Comment:
Did you add tests to see what happens if you create an account with a bad password?

Reply:
yep

Reply:
OK I saw that you do have such tests in the route tests. I don't think it would hurt to have an additional set of tests here, just to be safe.